### ISSUE TITLE
Does not work on iOS 14

### ISSUE THREAD

**Issue by panagiotisp95** [dashboardphilippines/react-webcam-barcode-scanner#13]

**Comment by msmurgui** [comment]
Any follow up on this?

**Comment by nfuller52** [comment]
It works on my iPad and iPhone using iOS 14.4

**Comment by shawalahmad123** [comment]
not working on any ios device in webview!
also not working in safari!

**Comment by nadezhdagorenok** [comment]
It doesn't work on Safari iPad -> iOS 14.6

**Comment by jim-pickering** [comment]
It works for me with Safari on iOS 14.7.1 - iPhone 12 Pro Max. I scanned a USAA 2D barcode and a FedEx 1D barcode with it.

**Comment by yuvalta** [comment]
Not working on iPhone 12 Pro - iOS 14.8 (Safari & Chrome)...

**Comment by victorosorioparis** [comment]
I do not think this is actually a problem with iOS, but with recent versions of Safari & Chrome and might not be related to this library.

Since July for example, Chrome is not allowing access to the webcam from websites that are not under HTTPS or localhost

**Comment by ivanpagac** [comment]
It works, but you have to set `HTTPS=true`, otherwise as @victorosorioparis mentioned, browser will block access to video html element. So instead of `npm start`, you type `HTTPS=true npm start` or adjust the `npm start` script.
